charts/observability/README.md

```diff
@@ -0,0 +1,23 @@
+# Medic Observability
+
+## Chart
```

Comment:
light preference to use [mermaid](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams) instead of ASCII, but not that big of a deal. If you do convert it, build it in [the live editor](https://mermaid.live/edit) and then paste in the resulting markdown - v. fast!

Reply:
I'm on the fence about this, mermaid if nice, but some of the consumers will be local terminal (`helm show readme <chart>` or locally checked out git repo) in which case I find it preferable to have plain ASCII. I vote that we keep it ASCII and reconsider mermaid if it becomes complex enough where ASCII is too limiting.

---
Here's an example how mermaid would look:
```mermaid
flowchart LR
subgraph Daemonset
NE[Node Exporter] -->|metrics| Vector
end
NL[Node Logs] -.->|logs| Vector
Opencost <-->|metrics| Prometheus
Vector -->|metrics| Prometheus
Prometheus -->|metrics| Grafana
Vector -.->|logs| Loki
Loki-.->|logs| Grafana
```